styles.css

```diff
@@ -56,6 +56,7 @@ html {
     text-shadow: 0 0.04em 0.04em rgba(255, 255, 255, 0.253);
     text-align: center;
     transition: all 0.2s;
+    cursor: pointer;
```

Comment:
- Mouse butona geldiğinde imlecin el şeklini alması için koyduğum küçük CSS talimatı.